docs/concepts/redirects.mdx

```diff
@@ -12,133 +12,156 @@ to configure it on Ory Cloud console. If you are interested in utilizing the SDK
+a FQDN. This will set the Login `Default Redirect URL` to
+`https://myapp.com/after/login` and the `Post-Password` to
+`https://myapp.com/after/login/password`.
 
 ![Browser Redirects URL Management Transformed](/images/browser-redirects/browser-redirects-url-management-2.png)
 
-### Verification, Recovery and Logout
+### Verification, recovery and logout
 
 :::warning
 
-The `Post-Recovery` redirect is not supported and will automatically redirect
-the user to the `Settings UI` URL. This implies that you should rely on the
-`Post-Settings` redirect URL for `Post-Recovery` flows as well.
+The `Post-Recovery` redirect isn't supported and will automatically redirect the
+user to the `Settings UI` URL.  
+Use the `Post-Settings` redirect of `Post-Recovery` flows.
 
 :::
 
-Each of these flows have only a single field, the `Default Redirect URL`. As
-with the Login, Registration and Settings configurations, the
-`Default Redirect URL` can be an FQDN or a path.
+Each of these flows has only a single field, the `Default Redirect URL`. As with
+Login, Registration and Settings configurations, the `Default Redirect URL` can
+be an FQDN or a path.
 
 ![Browser Redirects URL Management Transformed](/images/browser-redirects/browser-redirects-url-management-3.png)
 
-## Resetting/Updating Flows
+## Reset and update flows
 
-A redirect can be updated or reset by either changing the current redirects
-value or deleting the entry and clicking the "Update" button. All the flows can
-be reset at once by pressing the "Reset Flows" button which will prompt a
-confirmation box. Resetting all flows will reset all the fields back to the
-default value which is Ory Cloud [Managed User Interface](./managed-ui.mdx).
+A redirect can be updated or deleted by either changing the current redirects
+value or deleting the entry and clicking the **Update** button. All flows can be
+reset at once with the **Reset Flows** button which will prompt a confirmation
+box. Resetting all flows will reset all the fields back to the default value
+which is the Ory Cloud [Managed User Interface](./managed-ui.mdx).
 
 Once a flow has been reset or updated it will take effect immediately on **new**
 flows. Old flows that haven't expired yet won't be updated.
 
 ![Browser Redirects URL Reset All](/images/browser-redirects/browser-redirects-reset-all.png)
 
-## Troubleshooting Guide
+## Troubleshooting
 
-### Allow List
+### API clients
 
-The `Allow List` requires a valid URL to be set with a scheme (http or https).
+The browser redirects only work for regular browser requests.  
+If you are using an API Client for example AJAX, redirect the user to the right
+endpoint in the application, for example:
 
-![Browser Redirects URL Error Message](/images/browser-redirects/browser-redirects-error-messages-1.png)
+```ts
+          .then((res) => {
+            router.push('/<your-route>')
+          })
+```
 
-### Post Flow
+### `incorrect format`
```

Comment:
Let's avoid code in headings. Sometimes you just can't, but here your heading is JUST `code`. Try to think of a better one ;)